core/Console.php

```diff
@@ -34,11 +34,23 @@ public function __construct()
 
         $this->getDefinition()->addOption($option);
 
+        $option = new InputOption('xhprof',
+            null,
+            InputOption::VALUE_NONE,
+            'Enable profiling with XHProf'
+        );
+
+        $this->getDefinition()->addOption($option);
```

Comment:
Nah, you don't want to profile when development mode is enabled... it wouldn't cache, perform extra checks etc.